CONTRIBUTING.md

```diff
@@ -20,11 +20,13 @@ Let us know you're interested in working on a feature by filing a ticket in our
 
 ## Pull Requests and Code Review
 - Pull requests must be reviewed and approved by one non-author with write-permissions
+- If additional code changes are made after approval, another approval (from the same or another qualified person) is required.
 - A reviewer who has approved a PR but has not merged it should provide a comment (e.g. "This looks good, I'm just waiting for Travis to build", "This is a breaking change. I would like at least one more reviewer.")
 - An approved PR may be merged by the reviewer, the author, or another committer
+- Additional discussion and feedback by non-maintainers is heartily encouraged.
 
 ## Support
-If you are interested in working on the Blacklight plugin, but want guidance or support, please send an email to our [Blacklight-development mailing list](http://groups.google.com/group/blacklight-development) and we'll be glad to help.
+If you are interested in working on the Blacklight plugin, but want guidance or support, please send an email to our [Blacklight-development mailing list](http://groups.google.com/group/blacklight-development) and we'll be glad to help. The #blacklight channel on [code4lib Slack](https://code4lib.org/irc/) is another active forum for informal discussion and troubleshooting.
```

Comment:
What is "the Blacklight plugin"? Is that just "Blacklight"? 

How do you feel about people adding issues for support? Could be an opportunity to sunset the Google Group and just direct people to GH. 🤷‍♂️ Personally I feel like less places for communication has a lot of value, especially considering all the niceties GH has for interlinking and whatnot.